test/packages/EAStdC/CMakeLists.txt

```diff
@@ -2,11 +2,25 @@
 # Copyright (C) Electronic Arts Inc.  All rights reserved.
 #-------------------------------------------------------------------------------------------
 cmake_minimum_required(VERSION 3.1)
+project(EAStdC CXX)
 
 #-------------------------------------------------------------------------------------------
 # Library definition 
 #-------------------------------------------------------------------------------------------
-file(GLOB EASTDC_SOURCES "source/*.cpp")
+set(HEADER_DIR "include/EAStdC")
+set(SOURCE_DIR "source")
+set(EASTDC_SOURCES
+    ${HEADER_DIR}/EAAlignment.h
+    ${HEADER_DIR}/EADateTime.h
```

Comment:
Glob has the problem that any added, deleted or renamed files are not picked up when you build the project as long as you don't explicitly call CMake to regenerate your project / makefiles.
Even CMake itself discourages this practice: https://cmake.org/cmake/help/v3.1/command/file.html

Still, your call. ;)